test/controllers/SeriesDetailsControllerSpec.scala

```diff
@@ -67,47 +72,35 @@ class SeriesDetailsControllerSpec extends FrontEndTestHelper {
     }
```

Comment:
If you make the changes above to the controller where you use the `standardTypeAction`  instead of the `standardUserAction` then it will break the tests in this class because there's an extra API call you're not mocking.

I've got them working on a branch `change-user-action` if you want to have a quick look. You don't have to do it exactly like that but it might help.